packages/plugins/src/hardhat/deployments.ts

```diff
@@ -336,23 +339,34 @@ export const proposeChugSplashBundle = async (
   configUri: string,
   remoteExecution: boolean,
   ipfsUrl: string,
-  isUpgrade: boolean,
-  configPath: string
+  configPath: string,
+  spinner: ora.Ora,
```

Comment:
Including this parameter requires us to create a `spinner` instance each time we call this `proposeChugSplashBundle`, which isn't ideal if we aren't interested in logging information to the user. I got around this issue in `deployAllChugSplashConfigs` by setting this parameter to `spinner: ora.Ora = ora({ isSilent: true })`, which I think we should do here too.